src/bot/commands/AvatarCommand.ts

```diff
@@ -7,7 +7,7 @@ import UserCommand from './base/UserCommand';
 export default class AvatarCommand implements UserCommand {
   public readonly TRIGGERS = ['avatar'];
   public readonly NUMBER_OF_PARAMETERS = 1;
-  public readonly USAGE = 'Usage: !avatar [remove]';
+  public readonly USAGE = 'Usage: !avatar [remove|get|set]';
```

Comment:
There is no `!avatar get` command. this rather aims to say that you can use the command in multiple ways

```
!avatar (+ attachment)
!avatar remove
```

I want to rework and localize the usage messages anyways, so I'd just leave this as is for now